### PR TITLE
CarbonBlack Defense - context fix

### PR DESCRIPTION
**Pull request by roysagi**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/12688

## Description
Fixed an issue where the context output for ***cbd-get-alert-details*** was incorrect.
This is not a BC since it has always been that way.

image

### CODE REVIEW

**Comment by roysagi** [comment]
@amshamah419 - If you could please check the RN.

File: Packs/Legacy/Integrations/integration-CarbonBlackDefense.yml [hunk 1]
```diff
@@ -1336,65 +1334,65 @@ script:
       required: true
       description: The alert id
     outputs:
-    - contextPath: CarbonBlackDefense.GetAlertDetails.DeviceInfo.DeviceType
+    - contextPath: CarbonBlackDefense.GetAlertDetails.DeviceInfo.DeviceInfo.DeviceType
```
Comment (anara123) [inline Packs/Legacy/Integrations/integration-CarbonBlackDefense.yml:1337]: So it is `DeviceInfo.DeviceInfo` here?
double?

Comment (ronykoz) [reply Packs/Legacy/Integrations/integration-CarbonBlackDefense.yml:1337]: @anara123 yes, it's there since forever in the original integration

**Comment by amshamah419** [comment]
Looks good

**Comment by roysagi** [comment]
@anara123 @ronykoz - This needs to be force-merged since it changes context paths.

**Comment by ronykoz** [comment]
@roysagi let's fix the conflicts and i will merge it afterwards

**Comment by roysagi** [comment]
> @roysagi let's fix the conflicts and i will merge it afterwards

@ronykoz -Done

**Pull request merged**